File: app/src/main/java/tomaszkopacz/meetbam/presenters/RecyclerViewPresenter.java
package tomaszkopacz.meetbam.presenters;

import android.support.v7.widget.RecyclerView;

/**
 * Created by tomas on 07.04.2018.
 */

public interface RecyclerViewPresenter {

    void onItemBoundAtPosition(RecyclerView.ViewHolder holder, int position);
    int getItemCount();
}

File: app/src/main/java/tomaszkopacz/meetbam/presenter/RecyclerViewPresenter.kt
package tomaszkopacz.meetbam.presenter

import android.support.v7.widget.RecyclerView
import android.view.View

interface RecyclerViewPresenter {

    fun onItemBoundAtPosition(holder: RecyclerView.ViewHolder, position: Int)
    fun onItemClick(view: View)
    fun itemCount(): Int
}
